mobify/platform-scaffold
# Resize image to remove warning

Resize one of the iOS images to remove a compiler warning

## Changes
- Resize image

## How to test-drive this PR
- `cd native`
- `npm run deps`
- Open xcode
- Build
- See no compiler warnings related to images
